internal/cmd/operator-sdk/bundle/validate_test.go

```diff
@@ -18,6 +18,7 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/bundle/internal"
+	log "github.com/sirupsen/logrus"
 )
```

Comment:
We need to merge first: https://github.com/operator-framework/operator-sdk/pull/3808
You will need to rebase it with the master. So, could you please fix the order of the imports here. 
The sdk one should be kept and the end it will be: 

```
import (
	. "github.com/onsi/ginkgo"
	. "github.com/onsi/gomega"
	log "github.com/sirupsen/logrus"

       	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/bundle/internal"
)
``